core/bin/external_node/src/config/mod.rs

```diff
@@ -101,11 +101,11 @@ impl ConfigurationSource for Environment {
 /// This part of the external node config is fetched directly from the main node.
 #[derive(Debug, Deserialize)]
 pub(crate) struct RemoteENConfig {
-    pub bridgehub_proxy_addr: Option<Address>,
-    pub state_transition_proxy_addr: Option<Address>,
-    pub transparent_proxy_admin_addr: Option<Address>,
-    /// Should not be accessed directly. Use [`ExternalNodeConfig::diamond_proxy_address`] instead.
-    pub user_facing_diamond_proxy: Address,
+    pub l1_bridgehub_proxy_addr: Option<Address>,
+    pub l1_state_transition_proxy_addr: Option<Address>,
```

Comment:
IIRC this contract is used to read the preimage for the upgrade. The GW one should be used

Reply:
On a second thought, the L2 parts of both should be same, so reading just from L1 one is fine, it is just that only the SL one will contain the addresses of the new contracts (such as mailbox, verifier, etc), but these are not used anyway

Reply:
You're right, but it's a bug in eth watch, we should use different address there. I'll fix it separately

Reply:
> On a second thought, the L2 parts of both should be same, so reading just from L1 one is fine.

Eth watch is still broken, it calls L1 CTM with GW client.